cmd/mistryd/job.go

```diff
@@ -359,3 +353,55 @@ func (j *Job) BootstrapBuildDir(fs filesystem.FileSystem, log *log.Logger) (bool
 func (j *Job) RemoveBuildDir(fs filesystem.FileSystem, log *log.Logger) error {
 	return fs.Remove(j.ReadyBuildPath)
 }
+
+// ReadLogs returns the job's logs
+func (j *Job) ReadLogs() ([]byte, error) {
```

Comment:
I'd expect this method to just populate `j.BuildInfo.Log` instead of returning something.